src/pages/products/[productId].tsx

```diff
@@ -33,7 +34,10 @@ const ProductDescription = () => {
   }, [isRouterReady]);
 
   return isLoading ? (
-    <h1 className="text-center">Loading...</h1>
+    <div className="text-center">
```

Comment:
You can create a shared component for loding state.

Reply:
created shared isLoadingComponent